Runtime/Game/Resources/LootLockerConfig.cs

```diff
@@ -115,6 +154,69 @@ static void CreateConfigFile()
                 EditorPrefs.SetBool(configFileEditorPref, true);
             }
         }
+#if UNITY_EDITOR
+        static void WriteConfigToDisk()
+        {
+            //Check for an already existing persistent config file
+            string fileDirectory = "Assets/LootLockerSDK/Resources/Config/Editor/";
```

Comment:
We really should create these once statically I feel like and then just reuse the paths instead of typing it out like this every time.